media-capabilities/decodingInfo.any.js

```diff
@@ -291,6 +352,27 @@ promise_test(t => {
   }));
 }, "Test that decodingInfo rejects if the audio configuration contentType has one parameter that isn't codecs");
 
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'file',
+    audio: { contentType: 'audio/webm' },
+  }));
+}, "Test that decodingInfo rejects if the audio configuration contentType does not imply a single media codec but has no codecs parameter");
+
+promise_test(t => {
+  return promise_rejects_js(t, TypeError, navigator.mediaCapabilities.decodingInfo({
+    type: 'file',
+    audio: { contentType: 'audio/webm; codecs="vorbis, opus"' },
+  }));
+}, "Test that decodingInfo() rejects if the audio configuration contentType has a codecs parameter that indicates multiple audio codecs");
```

Comment:
```suggestion
}, "Test that decodingInfo rejects if the audio configuration contentType has a codecs parameter that indicates multiple audio codecs");
```

Reply:
Fixed in https://github.com/web-platform-tests/wpt/pull/49140/commits/56583ab14e348a6daa7614c50ddac447d5ec03e9.